builtin/int64_nonjs.mbt

```diff
@@ -177,7 +177,7 @@ pub fn Double::reinterpret_as_u64(self : Double) -> UInt64 = "%f64_to_i64_reinte
 ///|
 pub fn Double::reinterpret_as_uint64(self : Double) -> UInt64 = "%f64_to_i64_reinterpret"
 
-///|
+///| @alert deprecated "Use `UInt64::to_double` instead"
```

Comment:
UInt64 is not safe to represent in Double. I think `UInt64::to_double` is not a good name?

Reply:
It is as good / bad as `(int)1.0`.